dev/NavigationView/NavigationView.cpp

```diff
@@ -1533,6 +1514,17 @@ void NavigationView::UpdatePaneLayout()
                             const auto footersActualHeight = footerItemsRepeater.ActualHeight();
                             const auto menuItemsActualHeight = menuItems.ActualHeight();
 
+                            const auto paneFooterHeight = [this]() {
```

Comment:
For consistency, this should be called paneFooterActualHeight.

So this line will look better:
const auto footerGroupActualHeight = footersActualHeight + paneFooterHeight;
-->
const auto footerGroupActualHeight = footersActualHeight + paneFooterActualHeight;